Description: Don't enable logging by default
 By default it should "just work", silently. Logging exists for debugging
 purposes, so revert the upstream change introduced in 1.2.3.
Origin:      vendor
Author:      Didier Raboud <odyx@debian.org>
Last-Update: 2012-01-30
--- a/usb_modeswitch.conf
+++ b/usb_modeswitch.conf
@@ -16,4 +16,4 @@
 # Enable logging (results in a extensive report file in /var/log, named
 # "usb_modeswitch_<interface-name>" and probably others
 
-EnableLogging=1
+EnableLogging=0
